src/workers/parser/tests/__snapshots__/getOutOfScopeLocations.spec.js.snap

```diff
@@ -26,12 +26,6 @@ exports[`Parser.getOutOfScopeLocations should exclude non-enclosing function blo
 
 exports[`Parser.getOutOfScopeLocations should roll up function blocks 1`] = `
 "(1, 0) -> (1, 16)
-(4, 16) -> (6, 3)
-(8, 16) -> (10, 3)
-(12, 22) -> (14, 3)
-(16, 16) -> (18, 3)
-(20, 27) -> (22, 3)
-(24, 9) -> (26, 3)
```

Comment:
These are in fact the inner locations under the location enclosing the bp so shouldn't be out of scope